Update class Zombie

ImSchaeterwaeter opened the pull request:
-methods dropCurrency and attackGate added (not finished)
-added constructor call as you said

ImSchaeterwaeter committed:
Update class Zombie

-methods dropCurrency and attackGate added (not finished)
-added constructor call as you said

ShadowCreator250 commented on Zombie.java:
```diff
@@ -1,17 +1,35 @@
 		this.resistance = resistance;
 		this.speed = speed;
+		dropCurrency(this);
+		attackGate();
```
~~dropCurrency(this);~~
~~attackGate();~~

The method calls probably are not so well-placed in the constructor ;)
if you already want to add them, please define an act method (through override). ^^

ImSchaeterwaeter committed:
Update class Zombie

-new variable health
-resistance is directly involved in Tower classes
-empty act-method added
